nuitka/plugins/standard/standard.nuitka-package.config.yml

```diff
@@ -3039,6 +3050,23 @@
     patterns:
       - 'debug/sequences/*.lark'
       - 'debug/svd/*.zip'
+  implicit-imports:
```

Comment:
What is this about, can you point to where you have this from? Some of the module names seem excessive and/or wrong. These e.g. are not module names with a `-` in there typically.

Reply:
pyocd connect device need it